profiler/src/main/java/com/splunk/opentelemetry/profiler/LogsExporterBuilder.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+import static com.splunk.opentelemetry.profiler.Configuration.CONFIG_KEY_INGEST_URL;
+
+import com.splunk.opentelemetry.logs.InstrumentationLibraryLogsAdapter;
+import com.splunk.opentelemetry.logs.LogEntryAdapter;
+import com.splunk.opentelemetry.logs.LogsExporter;
+import com.splunk.opentelemetry.logs.OtlpLogsExporter;
+import com.splunk.opentelemetry.logs.OtlpLogsExporterBuilder;
+import com.splunk.opentelemetry.logs.ResourceLogsAdapter;
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
+import io.opentelemetry.sdk.resources.Resource;
+
+public class LogsExporterBuilder {
```

Comment:
needs to be public?

Reply:
also, why put this in the profiler package, rather than the logs package? If you did that, you wouldn't need to expose the properties on the exporter, right?

Reply:
nope, reduced it.

Reply:
I'm confused by this comment. If you put the builder into the logs package (along side what it builds), then you could move the test for it into the logs package, and the exporter could have it's accessors non-public. this class would be public still, which I think would be preferable to exposing things on the exporter itself.

Reply:
Unfortunately, I don't think this builder really exists other than to serve the profiling specific configurations....as it has profiling specific stuff in it.  As such, I think it should stay in the profiling package. There is also the `OtlpLogsExporterBuilder`, that's in the logs package.

Sorry about my terse/confusing comment...What I meant to say was that I made the change to reduce the access from public to package. 🤘🏻